mesheryctl/pkg/utils/formatter.go

```diff
@@ -18,3 +22,15 @@ func SetupLogrusFormatter() {
 	//log formatter for improved UX
 	log.SetFormatter(new(TerminalFormatter))
 }
+
+// Initialize Meshkit Logger instance
+func MeshkitLogger() (logger.Handler, error) {
+	log, err := logger.New("mesheryctl", logger.Options{
+		Format: logger.SyslogLogFormat,
+	})
+	if err != nil {
+		logrus.Error(err)
```

Comment:
We should get away from this logrus instance as well, right?

Reply:
Logrus needs to be completely wiped out, so best to replace these too.